index.bs

```diff
@@ -1617,8 +1617,21 @@ that are returned to the caller when a new credential is created, or a new asser
 
     dictionary AuthenticatorAttestationResponseJSON {
         required Base64URLString clientDataJSON;
-        required Base64URLString attestationObject;
+        required Base64URLString authenticatorData;
+        required long long publicKeyAlgorithm;
```

Comment:
Can `publicKeyAlgorithm` be positioned after the new `publicKey` below? I think it'd help to group the two together since they're related.

And reading the comment below for `publicKey`, would `publicKeyAlgorithm` also be missing if "pubKeyCredParams was used to negotiate a public-key algorithm that the user agent doesn't understand"? If so it might help emphasize that point by putting the two values togethere here.